doc/interfaces.scd

```diff
@@ -178,6 +178,11 @@ most common executors are:
 	The interface is a Virtual Extensible LAN (VXLAN) tunnel
 	endpoint.
 
+*wifi*
+	The interface is a Wi-Fi (IEEE 802.11) client interface.
+	Configuration of the WiFi client interface requires the
+	*wpa_supplicant* package to be installed.
```

Comment:
"if encryption should be used."

otherwise `iwconfig` is enough according to the executor. Maybe we should state that, too?